app/static/init.uv.js

```diff
@@ -0,0 +1,86 @@
+const script = document.getElementById("init-uv");
+const manifest_url = script.getAttribute("manifest_url");
+
+function initUniversalViewer() {
+  const data = {
+    manifest: manifest_url,
+    embedded: true,
+  };
+
+  const uv = UV.init("uv", data);
+  uv.on("configure", function ({ config, cb }) {
+    config.modules.centerPanel.options.usePdfJs = true;
+    config.modules.footerPanel.options = {
+      downloadEnabled: false,
+      embedEnabled: false,
+      fullscreenEnabled: false,
+      moreInfoEnabled: false,
+      shareEnabled: false,
+    };
+    cb({
+      options: {
+        footerPanelEnabled: false,
+        leftPanelEnabled: true,
+        rightPanelEnabled: false,
+        headerPanelEnabled: true,
+      },
+      pdfHeaderPanel: {
+        options: {
+          centerOptionsEnabled: false,
+        },
+      },
+    });
+  });
+}
+
+document.addEventListener("DOMContentLoaded", function () {
+  initUniversalViewer();
+});
+
+document.querySelectorAll(".govuk-tabs__tab").forEach((tab) => {
+  tab.addEventListener("click", function (event) {
+    if (this.getAttribute("href") === "#record-view") {
+      setTimeout(initUniversalViewer, 0);
+    }
+  });
+});
+
+// Function to remove the .attribution element
```

Comment:
why do we need to remove this element?

Reply:
Discussed on call. Also providing a screenshot of the attribution element which is removed. (Bottom left corner toolbar which contains the filename).

<img width="1063" alt="Screenshot 2024-08-05 at 12 05 04" src="https://github.com/user-attachments/assets/02a837cb-0cb2-470c-965b-b5ec4ed9ed52">